packages/openfeature-server-provider/src/version.ts

```diff
@@ -0,0 +1 @@
+export const VERSION = '0.0.5';
```

Comment:
I'm looking at getting release-please to update this using the extra-files config. I don't want to import from the package.json because it would mean that would need to be in the bundle, and also would require some more brittle IO. I think this is safer for a library running in the browser

Reply:
That should just be [here](https://github.com/spotify/confidence-openfeature-provider-js/blob/main/release-please-config.json#L22), right? Just not very certain if the file location is meant to be relative to the package or not.

Reply:
I think it's package, based off how the readme is accessed, also had to add the comment line for the generic updater. I'm trialing this in my test repo! :)

Reply:
verified this approach will work, and it is package scoped